includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -492,6 +492,25 @@ public function is_saved_cards_enabled() {
 		return 'yes' === $this->get_option( 'saved_cards' );
 	}
 
+	/**
+	 * Check if account is eligible for card present.
+	 *
+	 * @param null $empty_value - Default return value.
+	 * @return bool
+	 */
+	public function is_card_present_eligible( $empty_value = false ) {
+		try {
+			if ( ! $this->is_connected() ) {
```

Comment:
This does not seem like an error-prone piece of code. We may want to take it out of the `try` block?

Reply:
good hint. moved it outside the try block